src/odemis/gui/win/acquisition.py

```diff
@@ -874,6 +886,21 @@ def _resume_settings(self):
         self.streambar_controller.enable(True)
         self.streambar_controller.resume()
 
+    def _get_zstack_levels(self):
+        """
+        Calculate the zstack levels from the current focus position and zsteps value
+        :returns (list(float) or None) zstack levels for zstack acquisition
+        """
+        focus_value = self._main_data_model.focus.position.value['z']
+        # Clip zsteps value to allowed range
+        zsteps = numpy.clip(self.zstack_steps.current_value, MINZ, MAXZ)
+        if zsteps == 1:
+            return None
+        # Create focus zlevels from the given zsteps number
+        zlevels = list(numpy.linspace(focus_value - (zsteps / 2 * ZSTEP), focus_value + (zsteps / 2 * ZSTEP), zsteps,
+                                      endpoint=True))
```

Comment:
Default value of endpoint is True, so let's drop that, because it's just the normal behaviour.